tests/crypto/enclave/host/host.c

```diff
@@ -1,6 +1,11 @@
 // Copyright (c) Open Enclave SDK contributors.
 // Licensed under the MIT License.
 
+// Suppress warnings against using _open() instead of _sopen_s().
```

Comment:
Can you elaborate on the reason for this? Is this something that we're choosing not to fix or only deferring to fix?

Reply:
I have trouble locating the correct flags and modes for _sopen_s() but I will keep trying.

Reply:
Temporarily restored to the original version with C4996 disabled because `_sopen_s()` did not seem to work well. In my experiment, I changed

 - `return open(path, flags, mode);`

to

 - `_sopen_s(&fd, path, flags, _SH_DENYNO, mode);`

where I declared `int fd;` beforehand and returned it afterwards.